CHANGELOG.rst

```diff
@@ -6,7 +6,7 @@
 Synapse Changelog
 *****************
 
-v2.161.0 - 2024-02-02
+v2.161.0 - 2024-02-06
```

Comment:
Is this intentionally set to tomorrow?

Reply:
Yes I set it to today for burnin time :)